client/my-sites/site-settings/composing/index.jsx

```diff
@@ -10,44 +10,63 @@ import { connect } from 'react-redux';
 import Card from 'components/card';
 import DefaultPostFormat from './default-post-format';
 import AfterTheDeadline from './after-the-deadline';
-import { isJetpackSite, siteSupportsJetpackSettingsUi } from 'state/sites/selectors';
+import DateTimeFormat from '../date-time-format';
```

Comment:
I'd suggest avoiding relative paths, especially in site settings. Settings are known to be moved every now and then, so it's just easier if we import them by using absolute paths.